src/v1/destinations/iterable/types.ts

```diff
@@ -25,33 +27,43 @@ export type GeneralApiResponse = {
   failedUpdates?: FailedUpdates;
 };
 
-export type DestinationResponse = {
+export type IterableBulkApiResponse = {
   status: number;
   response: GeneralApiResponse;
 };
 
-export type ResponseParams = {
-  destinationResponse: DestinationResponse;
-  rudderJobMetadata: any[];
+type IterableBulkRequestBody = {
+  events?: any[];
+  users?: any[];
+};
+
+export type IterableBulkProxyInput = {
+  destinationResponse: IterableBulkApiResponse;
+  rudderJobMetadata: ProxyMetdata;
```

Comment:
shouldnt this be metadata array based on V1 request format?

Reply:
addressed